README.md

```diff
@@ -65,6 +67,39 @@ caffeinate -s pipenv run \
                 --app_gen_output_dir "$HOME/Desktop/multisuite_build_results/app_gen"
 ```
 
+You may also generate a project that matches your own project's dependency graph by using `--gen_type dot` parameter as well as supplying the location of the `dot` file that represents the graph:
```

Comment:
Could you link to the dot file format in wikipedia? https://en.wikipedia.org/wiki/DOT_(graph_description_language)

Also give an example command from buck (and cocoapods if you have it) to generate the dot graph? `buck query \"deps(target)\" --dot > file.gv`